### ISSUE TITLE
Microsoft.UI.Xaml.vcxproj.filters keeps getting entries added

### ISSUE THREAD

**Issue by jevansaks** [microsoft/microsoft-ui-xaml#821]
This is something that keeps coming up for devs. VS guys say this is a problem because there are project items pointing at things in $(OutDir). Their recommendation is to move those calculated items into an ItemGroup inside a target.

**Comment by StephenLPeters** [comment]
@llongley did your original fix not work?

**Comment by llongley** [comment]
No, it caused a runtime crash in debug bits, so it needed to be reverted.

**Comment by StephenLPeters** [comment]
Do we have a good idea of what needs to be done to fix the issue? Is this something that we should spend time on fixing?

**Comment by llongley** [comment]
The issue with the previous fix was that we weren’t able to retrieve the packaged theme resources correctly, if I’m recalling correctly.  I don’t know exactly what the correct fix would be – this is a fairly minor annoyance, so I haven’t had the bandwidth to look at it again to get a better fix.

**Comment by chingucoding** [comment]
During the past couple of months, I haven't noticed that VS wanted to add entries to MUXC.vcxproj.filters so maybe this issue has resolved itsself now?
